### PR TITLE
[OptApp] Restructure 19 - Standardized response routines

### PR DESCRIPTION
**Pull request by sunethwarna**
**📝 Description**
This PR adds `StandardizedObjective` and `StandardizedConstraint`

**🆕 Changelog**
- Added `StandardizedObjective`
- Added `StandardizedConstraint`

### CODE REVIEW

**Review by RezaNajian — DISMISSED** [review]

File: applications/OptimizationApplication/python_scripts/algorithms/standardized_objective.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+import KratosMultiphysics.OptimizationApplication as KratosOA
+from KratosMultiphysics.OptimizationApplication.utilities.optimization_problem import OptimizationProblem
+from KratosMultiphysics.OptimizationApplication.responses.response_routine import ResponseRoutine
+from KratosMultiphysics.OptimizationApplication.controls.master_control import MasterControl
+from KratosMultiphysics.OptimizationApplication.utilities.component_data_view import ComponentDataView
+
+class StandardizedObjective(ResponseRoutine):
+    """Standardized objective response function
+
+    This class creates instances to standardize any response function (make it a minimization problem).
```
Comment (Igarizza) [inline applications/OptimizationApplication/python_scripts/algorithms/standardized_objective.py]:
```suggestion
    This class transformed a user-given optimization problem into the standard format.
```

**Review by Igarizza — APPROVED** [review]
we can merge it

**Review by Igarizza — DISMISSED** [review]
we can merge it

**Review by RezaNajian — DISMISSED** [review]
Thanks Suneth !

**Review by Igarizza — APPROVED** [review]

**Pull request merged**